addons/website_forum/models/forum_post.py

```diff
@@ -868,3 +868,34 @@ def _search_render_results(self, fetch_fields, mapping, icon, limit):
             if with_date:
                 data['date'] = self.env['ir.qweb.field.date'].record_to_html(post, 'write_date', {})
         return results_data
+
+    def _get_related_posts(self, limit=5):
+        """Return at most a list of 5 posts related to the main post, based on tag (Jaccard) similarity."""
```

Comment:
Just to avoid people to have to check what it is if they don't know ?
```suggestion
        """Return at most a list of 5 posts related to the main post, based on tag 
        Jaccard similarity. It computes similarity of sets based on ratio of sets
        intersection divided by sets union (and thus varies from 0 to 1, 1 being
        identical sets)."""
```